truss/templates/triton/model/triton_model_wrapper.py

```diff
@@ -0,0 +1,80 @@
+import importlib
+import inspect
+import sys
+from pathlib import Path
+from typing import Dict
+
+import triton_python_backend_utils as pb_utils
+from shared.secrets_resolver import SecretsResolver
+from transform import transform_pydantic_to_triton, transform_triton_to_pydantic
+from utils.pydantic import inspect_pydantic_model
+from utils.signature import signature_accepts_keyword_arg
+
+
+class ModelWrapper:
+    def __init__(self, config: Dict):
+        self._config = config
+        self._model = None
+        self._input_type = None
+        self._output_type = None
+        self._input_type_fields = None
+        self._output_type_fields = None
+
+    def instantiate(self):
+        """
+        Instantiates the model class defined in the config.
+        """
+        model_directory_prefix = Path("/app/model/1/truss")
```

Comment:
What is this path? What is the `1` doing here?

Reply:
I'm also confused how this relates (or doesn't) to the model_repository path constructed from the triton_config in `TritonPythonModel`

Reply:
Please see [this](https://github.com/basetenlabs/truss/pull/537/files#diff-fcc20c8c80a4959f2008aa68b35e52e9d7708ec8bf6a4525d3a2ef7bbcaf53c8R176 ) comment for more information.